projects/components/src/filtering/filter/builder/filter-builder-lookup.service.test.ts

```diff
@@ -185,7 +185,7 @@ describe('Filter Builder Lookup service', () => {
       field: getTestFilterAttribute(FilterAttributeType.StringMap).name,
       operator: FilterOperator.ContainsKeyValue,
       value: ['myKey', 'myValue'],
-      userString: 'String Map Attribute CONTAINS_KEY_VALUE myKey:myValue',
+      userString: 'String Map Attribute.myKey CONTAINS_KEY_VALUE myValue',
```

Comment:
This is how a user will use the filter chip autocomplete though. Not sure I understand why this is wrong.

Reply:
It's not wrong, I just think it's a lot weirder to have something like `My Pretty Display Name.somePath` vs `name.somePath` - it feels like we're mixing two things.

Reply:
Yeah, I agree it feels like we are mixing two different worlds but not sure what the alternative is since we expect users to work with display names. Once upon a time we were only using the attribute names here, but were asked to switch to display names. I can see both sides.